### ISSUE TITLE
Franz Crashes my system

### ISSUE THREAD

**Issue by Konst0** [meetfranz/franz#1262]
**Describe the bug**
Installed Franz 5 beta 24 on a newly reinstalled windows 10 version 1809. Whenever I open Franz, it crashes my computer. Everything stops and I have to manually shut down the computer with the power switch.

When Im back up running again, when launching Franz, nothing happens. The service is running in the taskmanager (multiple actually) but nothing shows in the minimized application tray.

If I remove all services in task manager, then open Franz again, nothing happens. 

If I uninstall Franz, remove every file related to Franz from my computer, then install Franz again, the whole system crashes.

I have used Franz for a long time, and this issue is quite strange, especially since this is a fresh install of windows.10.

**Expected behavior**
For Franz to open and work normally.

**Desktop (please complete the following information):**
 - Windows 10 version 1809 (fresh install)
 - Firefox

**Comment by kachrahp** [comment]
I am having the same problem, Its running in task manager but I can't actually see the window. I have tried uninstalling and reinstalling franz and same thing happens.

However this isn;t a fresh install, I remember franz asking me to update the service yesterday, and worked but this morning its not at all.

Have windows 10 64 bit. And using Chrome.

**Comment by Konst0** [comment]
This has been an annoying journey, right now things seem to work properly. 

The issue I had was not essentially linked to Franz, however, Franz was a trigger to cause the freezes. During the last days I've talked to 4 different windows representatives to get it to work and everyone have given me different answer and who have clumsily used remote control to my computer and switching off and on options in the msconfig window and process window.

The solution to my problem was to reinstall graphic drivers, however, I did this plenty times before and it didnt make any changes. I first downloaded my graphic drivers from Nvidia driver download, which didnt work. Then I used the Geforce Nvidia driver download page which has the same naming and numbering of graphic card models, but when downloading from there it now seem to have fixed the problem.

When reinstalling Franz, one must remove all temp and prefetch files on the system, and all the folders of Franz, because the uninstall wont remove these.

**These are the locations**
- C:\Windows\Prefetch (when at this location, just write Franz on your keyboard and it should find the correct files)
- C:\Users\YOUR USERNAME\AppData\Local\Programs\franz
- C:\Users\YOUR USERNAME\AppData\Roaming
- C:\Users\YOUR USERNAME\AppData\Local\Packages\Microsoft.MicrosoftEdge_8wekyb3d8bbwe\AC\#!001\MicrosoftEdge\User\Default\DOMStore\E1KBDE96

To more easily find these files, I use a program called Everything Search from Voildtool. To be honest, this is how windows search should have been,
https://www.voidtools.com/

I hope this helps for readers with the same issue. If anything is unclear, just write a comment.
